### PR TITLE
Add 3.3.4 migration instructions

### PR DESCRIPTION
**Pull request by rschwammborn**
Manual migration information, as seen in [changelog](https://github.com/mapbender/mapbender/blob/master/CHANGELOG.md).

### CODE REVIEW

File: en/installation/migration.rst [hunk 1]
```diff
@@ -8,6 +8,26 @@ Migration Guide
...
+Migration to Mapbender 3.3.4
+****************************
+
+* Please update the database schema. Run:
+
+.. code-block:: bash
+
+    $ app/console doctrine:schema:update --force
+
+* Next, you need to add a CSRF token in ``security.yml``. Configure the ``form_login`` settings as follows:
```
Comment (Phocacius) [inline en/installation/migration.rst:20]: the line does not by itself add a CSRF token, but configures them to be used, so the wording is a bit inaccurate.

Suggestion: You need to configure CSRF tokens to be used during login by adding the following line in `security.yml`.

If possible (I never used rst myself), print  the last line as bold text.

File: en/installation/migration.rst [hunk 2]
```diff
@@ -8,6 +8,26 @@ Migration Guide
...
+
+.. code-block:: yaml
+
+    form_login:
+        check_path: /user/login/check
+        login_path: /user/login
+        csrf_token_generator: security.csrf.token_manager
+
+* For productive environments, it is important to install an SSL certificate. After that, set the ``parameters.cookie_secure`` variable in your ``parameters.yml`` to ``true``. This ensures that the Login cookie is only transmitted over secure connections.
```
Comment (Phocacius) [inline en/installation/migration.rst:29]: Also add this in the configuration section of the regular user guide (not only the migration). Doesn't hurt to mention it in the migration as well though :)

Comment (rschwammborn) [reply en/installation/migration.rst:29]: Right! Added SSL info to yaml.rst.

File: en/installation/migration.rst (hunk 1 above)
Comment (rschwammborn) [reply en/installation/migration.rst]: I took your text suggestion and added a **reference** above the code section, see my commit below.

**Review by Phocacius — APPROVED** [review]

File: en/customization/yaml.rst [hunk 3]
```diff
@@ -8,35 +8,12 @@ The following Configuration files are under application/app/config.
...
-                path:      "%database_path%"                # %database_path%, path to the file of the SQLite database. If you don't use a SQ-lite database, write (~) or ``null``.
-                user:      "%database_user%"                # User name for database connection.
-                password:  "%database_password%"            # Password.
-                charset:    UTF8                            # Coding of the database.
-                logging:   "%kernel.debug%"                 # Option, SQLs won't be logged (standard: %kernel.debug%). `More information: <http://www.loremipsum.at/blog/doctrine-2-sql-profiler-in-debugleiste>`_.
-                profiling: "%kernel.debug%"                 # Profiling SQL requests. This option can be turned of in production. (standard: %kernel.debug%)
-
+To configurate the database the files config.yml and parameters.yml are needed. The file config.yml contains placeholders for variables, which are specified in the file parameters.yml. For the default database definiton set in ``config.yml``, see below.
```
Comment (astroidex) [inline en/customization/yaml.rst:16]: see german version. Bitte auch anpassen

File: en/installation/migration.rst [hunk 4]
```diff
@@ -8,6 +8,26 @@ Migration Guide
...
 
 
+Migration to Mapbender 3.3.4
+****************************
+
+* Please update the database schema. Run:
+
+.. code-block:: bash
+
+    $ app/console doctrine:schema:update --force
```
Comment (astroidex) [inline en/installation/migration.rst:18]: $ kann weggelassen werden

File: en/installation/migration.rst [hunk 5]
```diff
@@ -8,6 +8,26 @@ Migration Guide
...
+
+* Next, you need to configure CSRF tokens to be used during login by adding **csrf_token_generator: security.csrf.token_manager** to the form_login section in ``security.yml`` as follows:
```
Comment (astroidex) [inline en/installation/migration.rst:20]: If you use the acutual code the section is already added to the security.yml file

**Review by astroidex — CHANGES_REQUESTED** [review]

**Review by astroidex — APPROVED** [review]

**Pull request merged**